tests/configmap/manifests/ksp-unannotated-allow.yaml

```diff
@@ -11,26 +11,38 @@ spec:
   selector:
     matchLabels:
       container: unannotated
-  process:
+  file:
     matchDirectories:
-    - dir: /bin/ # required to change root to user1
+    - dir: /etc/
       recursive: true
-    - dir: /usr/bin/ # used in changing accounts
+    - dir: /lib/x86_64-linux-gnu/
       recursive: true
-  file:
+    - dir: /credentials/
     matchPaths:
-    - path: /root/.bashrc # used by root
-    - path: /root/.bash_history # used by root
     - path: /dev/tty
-    matchDirectories:
-    - dir: /credentials/
-    - dir: /etc/ # used by root (coarse-grained way)
-      recursive: true
-    - dir: /proc/ # used by root (coarse-grained way)
-      recursive: true
-  action:
+    - path: /etc/ld.so.cache
+    - path: /etc/passwd
+    - path: /lib/terminfo/x/xterm
+    - path: /pts/0
+    - path: /pts/ptmx
+    - path: /root/.bash_history
+    - path: /root/.bashrc
+    - path: /bin/bash
+      readOnly: true
+    - path: /bin/cat
+      readOnly: true
+    - path: /bin/sh
+      readOnly: true
+    - path: /usr/bin/groups
+      readOnly: true
+  process:
+    matchPaths:
+    - path: /bin/bash
+    - path: /bin/cat
+    - path: /bin/sh
+    - path: /usr/bin/groups
```

Comment:
This change should not be required since previously we included `/bin/*` and `/usr/bin/*` in the the spec.